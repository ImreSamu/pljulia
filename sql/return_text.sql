CREATE FUNCTION julia_text()
RETURNS TEXT AS $$
    "text"
$$ LANGUAGE pljulia;
SELECT julia_text();
DROP FUNCTION julia_text();
