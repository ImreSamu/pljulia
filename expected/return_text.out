CREATE FUNCTION julia_text()
RETURNS TEXT AS $$
    "text"
$$ LANGUAGE pljulia;
SELECT julia_text();
 julia_text 
------------
 text
(1 row)

DROP FUNCTION julia_text();
